components/search_results/search_results.jsx

```diff
@@ -197,23 +196,32 @@ class SearchResults extends React.Component {
             );
         } else if (this.props.isFlaggedPosts && noResults) {
             ctls = (
-                <div className='sidebar--right__subheader a11y__section'>
-                    <FlagPostSearchHint
-                        dataRetentionEnableMessageDeletion={this.props.dataRetentionEnableMessageDeletion}
-                        dataRetentionMessageRetentionDays={this.props.dataRetentionMessageRetentionDays}
+                <div
+                    className={classNames(['sidebar--right__subheader a11y__section',
+                        {'sidebar-expanded': this.props.isSideBarExpanded && noResults,
+                        }])}
```

Comment:
nit: Multiline, same case with a couple of other instances in the file. Is this an issue with your formatter?